src/EventStore.Core/Messages/ClientMessage.cs

```diff
@@ -1751,6 +1751,110 @@ public enum ScavengeResult {
 				Stopped,
 				InvalidScavengeId
 			}
+		}*/
+		
+		[DerivedMessage]
+		public partial class ScavengeDatabaseGetResponse : Message {
+			public readonly Guid CorrelationId;
+			public readonly ScavengeResult Result;
+			public readonly string ScavengeId;
+
+			public ScavengeDatabaseGetResponse(Guid correlationId, 
+				ScavengeResult result, string scavengeId) {
+				CorrelationId = correlationId;
+				Result = result;
+				ScavengeId = scavengeId;
+			}
+
+			public override string ToString() {
+				return String.Format("Result: {0}, ScavengeId: {1}", Result, ScavengeId);
+			}
```

Comment:
could consider using string interpolation

```suggestion
			public override string ToString() =>
				$"Result: {Result}, ScavengeId: {ScavengeId}");
```

Reply:
Thank you for the suggestion Tim. Made the changes